### PR TITLE
fix: Cosmetic UI issues when viewing miner stats #282

### PR DESCRIPTION
**Pull request by zhbh**
[fix]: Cosmetic UI issues when viewing miner stats #282

### CODE REVIEW

File: lib/page/home_page/gateway/view_all_page/component/bar_chart.dart [hunk 1]
```diff
@@ -31,11 +31,32 @@ class DDBarChart extends StatefulWidget {
...
   Widget build(BuildContext context) {
+    Size _screenSize = MediaQuery.of(context).size;
+
+    Widget _tooltipShow() {
+      if (_indexTappedBar != -1 &&
+          widget.tooltipData.isNotEmpty &&
+          _indexTappedBar < widget.tooltipData.length) {
+        if ((_position.dx + (widget.tooltipData[_indexTappedBar].length * 2) + 40) <
+            _screenSize.width) {
+          return Text('${widget?.tooltipData[_indexTappedBar] ?? 0}',
```
Comment (miljan-mxc) [inline lib/page/home_page/gateway/view_all_page/component/bar_chart.dart:47]: why is 0 default value? if there is no data, don't show tooltip

File: lib/page/home_page/gateway/view_all_page/component/bar_chart.dart [hunk 2]
```diff
@@ -31,11 +31,32 @@ class DDBarChart extends StatefulWidget {
...
+              style: FontTheme.of(context).big());
+        } else {
+          return Text(
+              '${widget?.tooltipData[_indexTappedBar] ?? 0}'
```
Comment (miljan-mxc) [inline lib/page/home_page/gateway/view_all_page/component/bar_chart.dart:51]: same here

Comment (zhbh) [reply lib/page/home_page/gateway/view_all_page/component/bar_chart.dart:51]: ok

File: lib/page/home_page/gateway/view_all_page/component/bar_chart.dart (hunk 1 above)
Comment (zhbh) [reply lib/page/home_page/gateway/view_all_page/component/bar_chart.dart:47]: ok

**Review by miljan-mxc — APPROVED** [review]

**Pull request merged**